Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -2043,6 +2042,292 @@ public static void RegisterPlayerProgression(string progressionKey, Action<LootL
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerPlayerHeroResponse</param>
+        public static void GetOtherPlayersDefaultHeroBySteamID64(int steamId, Action<LootLockerPlayerHeroResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerPlayerHeroResponse>());
+                return;
+            }
+
+            LootLockerAPIManager.GetOtherPlayersDefaultHeroBySteamID64(steamId, onComplete);
+
+        }
+
+        /// <summary>
+        /// Update the name of the hero with the specified id and/or set it as default for the current player
+        /// </summary>
+        /// <param name="heroId">Id of the hero</param>
+        /// <param name="name">The new name for the hero</param>
+        /// <param name="isDefault">Should this hero be the default hero?</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerPlayerHeroResponse</param>
+        public static void UpdateHero(string heroId, string name, bool isDefault, Action<LootLockerPlayerHeroResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerPlayerHeroResponse>());
+                return;
+            }
+
+            LootLockerGetRequest lootLockerGetRequest = new LootLockerGetRequest();
+
+            lootLockerGetRequest.getRequests.Add(heroId);
+
+            LootLockerUpdateHeroRequest data = new LootLockerUpdateHeroRequest();
+            data.name = name;
+            data.is_default = isDefault;
+
+
+            LootLockerAPIManager.UpdateHero(lootLockerGetRequest, data, onComplete);
+        }
+
+        /// <summary>
+        /// Remove the hero with the specified id from the current players list of heroes.
+        /// </summary>
+        /// <param name="heroID">HeroID Id of the hero</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerPlayerHeroResponse</param>
+        public static void DeleteHero(int heroID, Action<LootLockerPlayerHeroResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerPlayerHeroResponse>());
+                return;
+            }
+
+            LootLockerAPIManager.DeleteHero(heroID, onComplete);
+        }
+
+        /// <summary>
+        /// List Asset Instances owned by the specified hero
+        /// </summary>
+        /// <param name="heroID">HeroID Id of the hero</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerInventoryResponse</param>
+        public static void GetHeroInventory(int heroID, Action<LootLockerInventoryResponse> onComplete)
```

Comment:
This is missing from the Unreal Implementation as well, but this call is paginated